config/routes.rb

```diff
@@ -53,7 +53,7 @@
       get '/processed', action: :processed, as: 'processed_visits'
       get '/print_visits', action: :print_visits, as: 'print_visits'
       get '/search', action: :search, as: :search
-      post '/switch_estate', action: :switch_estate, as: 'switch_estate'
+      post '/switch_estates', action: :switch_estates, as: 'switch_estates'
```

Comment:
Using ```resources``` even if it means a having a controller with only one action isn't a big deal and encourages separation of concerns. I'm weary to see fatter controller than what we have now.